### PR TITLE
Ig updates

### PR DESCRIPTION
**Pull request by ptran265**

### CODE REVIEW

File: input/images/image2.png
Comment (mickohanlon23) [inline input/images/image2.png:1]: New picture dimensions are slightly different, causing the image to appear disproportionate in the rendered IG. We should update the image to use the same dimensions, or update the height and width specified in the following line: https://github.com/CancerScreeningCDS/crcsm-cds/blob/igUpdates/input/pagecontent/2_background.md?plain=1#L134.

File: input/images/highlevelcrc.png
Comment (mickohanlon23) [inline input/images/highlevelcrc.png:1]: High level flow diagram is a bit difficult to actually read. Is there anything we can do to adjust this? Maybe take/generate a high resolution screenshot of the flow diagram?

File: input/pagecontent/4_examples.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-### TestCases
+### Test Cases
+This section lists the test cases that are generated for the [CDS Level 3](https://www.hl7.org/fhir/uv/cpg/STU2/documentation-approach-06-01-levels-of-knowledge-representation.html){:target="_blank"} (L3) logical pathways for colorectal cancer guidelines for screening and management. The test case templates are generated in advance of developing computable L3 logic that implements the CDS L2 logical pathways. The test cases provide unit tests, both negative and positive, for each of the clinical logic pathways. These test cases consist of synthetic electronic health records for a hypothetical patient as well as the corresponding expected CDS outputs. Listed below are the synthetic electronic health record snippets for each of these test patient fixtures. See the [“Testing and Validation” Section](#cds-testing--validation){:target="_blank"} for more details.
```
Comment (mickohanlon23) [inline input/pagecontent/4_examples.md]:
```suggestion
This section lists the test cases that are generated for the [CDS Level 3](https://www.hl7.org/fhir/uv/cpg/STU2/documentation-approach-06-01-levels-of-knowledge-representation.html){:target="_blank"} (L3) logical pathways for colorectal cancer guidelines for screening and management. The test case templates are generated in advance of developing computable L3 logic that implements the CDS L2 logical pathways. The test cases provide unit tests, both negative and positive, for each of the clinical logic pathways. These test cases consist of synthetic electronic health records for a hypothetical patient as well as the corresponding expected CDS outputs. Listed below are the synthetic electronic health record snippets for each of these test patient fixtures. See the [“Testing and Validation” Section](3_specification.html#cds-testing--validation){:target="_blank"} for more details. 
```

Testing and Validation link should be updated to point to specific section within Specification page.

File: input/images/highlevelcrc.png
Comment (mrnosal) [reply input/images/highlevelcrc.png:1]: These diagrams just have awkward layouts to begin with. Too much tiny text in the gray boxes, too much awkward whitespace and weird alignment of elements. At least bump up the number of pixels (the CDS diagram is at 2x resolution).

File: input/pagecontent/4_examples.md (hunk 1 above)
Comment (mickohanlon23) [reply input/pagecontent/4_examples.md]: I addressed this in commit https://github.com/CancerScreeningCDS/crcsm-cds/pull/13/commits/5a419918aa6c556ca17b40d1b0c891663fe9ba9c.

File: input/images/image2.png
Comment (mickohanlon23) [reply input/images/image2.png:1]: I addressed this in commit https://github.com/CancerScreeningCDS/crcsm-cds/pull/13/commits/1804b2a8e6c24f156783e7782f4f67fee89472db.

File: input/images/highlevelcrc.png
Comment (mickohanlon23) [reply input/images/highlevelcrc.png:1]: What is the best way to bump up the number of pixels?

Comment (mickohanlon23) [reply input/images/highlevelcrc.png:1]: I resolved this in the latest 3 commits. I:
1. Added the original high-resolution version of the image
2. Added a "Figure X: ..." caption to the High-Level flow diagram image
3. Fixed incorrectly numbered figure captions throughout the narrative IG content.

**Review by mickohanlon23 — APPROVED** [review]
All issues have been resolved by the latest commits. Approved! @ptran265 can you take one last look, specifically at the newly updated figures and figure captions, to ensure that the IG is building correctly on your end. Once you confirm this, you can go ahead and merge this in.

**Pull request merged**